modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ManagementToolbar/index.tsx

```diff
@@ -164,10 +176,10 @@ export function ManagementToolbar({
 
 							{isApproved !== undefined && !isApproved && (
 								<ClayButton
-									disabled={!hasPublishPermission}
+									disabled={disabled}
```

Comment:
Hey @vngoliveira, here we also need `hasPublishPermission` validation, so it can be: 

```tsx
disabled={!hasUpdatePermission || disabled}
```